test_team_2.py

```diff
@@ -12,6 +12,18 @@ def test_anonymize_text(self):
 
     def test_crypto(self):
         """Test crypto functionality"""
+        #positive test case, should replace the address with <CRPYTO> 
+        prefix = ['3', '1']
+        end = ['QJmV3qfvL9SuYo34YihAf3sRCW3qSinyC', 'C7zdTfnkzmr13HfA2vNm5SJYRK6nEKyq8']
+        #use for loop to create addresses that won't flag detection systems
+        for index, value in enumerate(prefix):
+            expected = 'My bitcoin wallet address is <CRYPTO>'
+            actual = 'My bitcoin wallet address is ' + value + end[index]
+            self.assertEqual(anonymize_text(actual, ['CRYPTO']), expected)
+            #negative test case 1: Will not be replaced due to invalid format
```

Comment:
These three lines should be outside of the for loop - like this:
```
        for index, value in enumerate(prefix):
             expected = 'My bitcoin wallet address is <CRYPTO>'
             actual = 'My bitcoin wallet address is ' + value + end[index]
             self.assertEqual(anonymize_text(actual, ['CRYPTO']), expected)

         #negative test case 1: Will not be replaced due to invalid format
         invalid = 'My bitcoin wallet address is 24qar0srrr7xfkvy5l643lydnw9re59gtzzwf5mdq'
         self.assertEqual(anonymize_text(invalid, ['CRYPTO']), invalid)
```

Reply:
Thank you, I did not realize I was running the invalid on the positive test case loop.